akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -461,7 +461,7 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
 
     log.info(
       "Remoting started with transport [Artery {}]; listening on address [{}] with UID [{}]",
-      settings.Transport, localAddress.address, localAddress.uid)
+      settings.Transport, bindAddress.address, bindAddress.uid)
```

Comment:
I don't think that is what we want, but perhaps it should check if bind and local address is the same, and add info about " bound to [{}]" in that case?

Local address is important because the actor system will not accept messages unless the local address is the destination while the bind address is a machine-local detail.

Reply:
+1 to logging both if they are different

Reply:
I like the idea of login both. Updated it. Thanks!